apps/docs/src/app/core/component-docs/table/examples/table-example.component.html

```diff
@@ -13,8 +13,16 @@
             <td fd-table-cell>
                 <a fd-link href="#">{{ row.column1 }}</a>
             </td>
-            <td fd-table-cell>{{ row.column2 }}</td>
-            <td fd-table-cell>{{ row.column3 }}</td>
+            <td fd-table-cell>
+                <p fd-table-text style="max-width: 250px;" [noWrap]="true">
```

Comment:
We can even move `max-width` to be a prop and then the user will avoid writing `style="..."`